package.json

```diff
@@ -62,7 +64,9 @@
         "morgan": "^1.10.0",
         "morgan-json": "^1.1.0",
         "multer": "^1.4.5-lts.1",
+        "multer-s3": "^3.0.1",
```

Comment:
@valeriagrazzini And are you sure this one should be added too? I dont see any logic in this PR using it and our current S3 client works just fine right?